purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesFactory.kt

```diff
@@ -77,7 +75,7 @@ internal class PurchasesFactory(
 
         with(configuration) {
             val finalStore = if (
-                apiKeyValidationResult == APIKeyValidator.ValidationResult.SIMULATED_STORE && isSimulatedStoreEnabled()
+                apiKeyValidationResult == APIKeyValidator.ValidationResult.SIMULATED_STORE
             ) {
                 Store.UNKNOWN_STORE // We should add a new store when we fully support the simulated store.
```

Comment:
I wonder, should we add a new store now for these purchases? I think that was added in iOS already, but we didn't do it in android to avoid the new APIs until we were ready.